src/server/room-controller.spec.js

```diff
@@ -221,8 +231,13 @@ describe('room-controller', () => {
           }
         };
 
+        const lessons = [];
+
         beforeEach(async () => {
           roomService.isRoomOwnerOrMember.resolves(true);
+          roomService.isRoomOwnerOrMember.resolves(true);
```

Comment:
This just repeats the line above, typo I guess.

Reply:
Yup, leftover, removed, thanks!